src/mode/modeHandler.ts

```diff
@@ -81,12 +81,16 @@ export class ModeHandler implements vscode.Disposable {
         }
     }
 
-    private setupStatusBarItem(text : string) : void {
+    private setupModeStatusBarItem(text : string) : void {
```

Comment:
Can we consolidate these functions? `setupModeStatusBarItem` and `setupStatusBarItem`?